aws/offsite/aurora-snapshots-fargate/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:18.04
+FROM ubuntu:20.04
```

Comment:
I'm not entirely sure how this Dockerfile is used, or how to test this change. Anyone else know?